swagger-ui/index.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
```

Comment:
Do you need this file?

Reply:
Yes, this file is need for generate swagger-ui bundle